unitary/examples/quantum_rpg/encounter.py

```diff
@@ -0,0 +1,39 @@
+import io
+from typing import Optional, Sequence
+
+import random
+
+import unitary.examples.quantum_rpg.battle as battle
+import unitary.examples.quantum_rpg.qaracter as qaracter
+from typing import Sequence
+
+
+class Encounter:
+    """Container class for specifying encounters.
+
+    Useful in specifying an adventure in quantum RPG.
+    """
+
+    def __init__(
+        self,
+        enemies: Sequence[qaracter.Qaracter] = (),
+        probability: float = 1.0,
+        description: Optional[str] = None,
+    ):
+        self.enemies = enemies
+        self.probability = probability
+        self.description = description
+
+    def trigger(self) -> bool:
```

Comment:
nit. the name `trigger` sounds like this will trigger the encounter itself. Maybe give it a bool name like `will_trigger`?

Reply:
Done.